mleap-xgboost-java/src/main/java/biz

```diff
@@ -0,0 +1 @@
+../../../../vendor/xgboost-predictor-java/xgboost-predictor/src/main/java/biz
```

Comment:
This shows up kind of strange in github, but this is a symlink to the xgboost-predictor-java submodule src folder containing the predictor code.